### DISCUSSION TITLE
Please make Bss_FastOrder compatible with Breeze

### DISCUSSION THREAD

**Discussion by pabodah** [breezefront/community#128] [3. Integrations]
Hello,

The module Bss_FastOrder - [https://bsscommerce.com/magento-2-wholesale-fast-order-extension.html)](https://bsscommerce.com/magento-2-wholesale-fast-order-extension.html) is not compatible.
Please check.

**Comment by dimdimich** [comment]
Can you please try contacting BSS and ask to consider adding Breeze integration? Right now we can't work on integration of this module.
